packages/react-components/react-storybook-addon/tsconfig.spec.json

```diff
@@ -5,5 +5,13 @@
     "outDir": "dist",
     "types": ["jest", "node"]
   },
-  "include": ["**/*.spec.ts", "**/*.spec.tsx", "**/*.test.ts", "**/*.test.tsx", "**/*.d.ts"]
+  "include": [
+    "**/*.spec.ts",
+    "**/*.spec.tsx",
+    "**/*.test.ts",
+    "**/*.test.tsx",
+    "**/*.d.ts",
+    "./src/testing/**/*.ts",
```

Comment:
shouldn't this be added only if such a setup exists or it's default ? if it's default it should be also always excluded from `tsconfig.lib.json`

Reply:
Yup you're right, this was an oversight in the implementation. PR to address this: #25493